simtools/util/names.py

```diff
@@ -748,3 +755,44 @@ def camera_efficiency_log_file_name(site, telescope_model_name, zenith_angle, la
     name += f"_{label}" if label is not None else ""
     name += ".log"
     return name
+
+
+def get_telescope_type(telescope_name):
+    """
+    Guess telescope type from name. Types are "LST", "MST", "SST", "SCT".
+    Parameters
+    ----------
+    telescope_name: str
+        Telescope name
+    Returns
+    -------
+    str
+        Telescope type.
+    """
+
+    _class, _ = split_telescope_model_name(telescope_name)
+    try:
+        if _class[0:3] in ("LST", "MST", "SST", "SCT"):
+            return _class[0:3]
+
+    except IndexError:
+        pass
+
+    return ""
+
+
+def get_corsika_telescope_data_dict():
+    io_handler = simtools.io_handler.IOHandler()
+    io_handler.set_paths(output_path="./simtools-output/", data_path="./data/")
```

Comment:
These are configuration parameters, paths shouldn't be hard coded.

Reply:
I am having a hard time adjusting to the configuration parameters. should I initialize here the configurator to load the default paths?

Reply:
But why should a method called `get_corsika_telescope_data_dict()` return a dict with the observation level for `North` only? What if you want to do that for Paranal?

@orelgueta - do we have the `corsika_sphere_radius` in the database? This is all what is needed here.

I also think that `names.py` is not the correct place to put this method. This is about names....

Reply:
I made those comments as well and explained my suggestions for them (not much different from yours), see below.

It is not in the DB yet. Generally we don't have CORSIKA in the DB yet as I wanted Victor to finish his tool first and then we can see what exactly is needed. So for now we use this yaml file which has everything.